src/main/java/org/edumips64/ui/swing/DineroFrontend.java

```diff
@@ -137,10 +144,110 @@ public void keyReleased(KeyEvent e) {
       }
     });
 
-    JButton browse = new JButton("Browse...");
+		String[] level = {"1", "2", "3", "4", "5"};
```

Comment:
Please use indentation consistent with the rest of the file (2 spaces to indent). Otherwise it becomes hard to follow the flow of the file if it uses mixed indentation.

Reply:
Tried to fix the indentation, will get back to this part again once the rest is done.